snmp/assets/configuration/spec.yaml

```diff
@@ -552,3 +552,9 @@ files:
       - template: instances/default
         overrides:
           service.hidden: true
+      - name: collect_topology
```

Comment:
We should have this higher on the config, at least before `instances/default`

Reply:
https://github.com/DataDog/integrations-core/pull/17407/commits/73c0f9cb7fa74cd2ed8c3780a98a17e27e0d92cf is that ok for you ?